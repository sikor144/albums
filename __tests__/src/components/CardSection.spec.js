import React from 'react';
import renderer from 'react-test-renderer';

import CardSection from '../../../src/components/CardSection';

describe('CardSection component', () => {
  it('renders component correctly', () => {
    const component = renderer.create(
      <CardSection />,
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
