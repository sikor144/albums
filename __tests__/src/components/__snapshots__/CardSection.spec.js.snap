// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSection component renders component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderBottomWidth": 1,
      "borderColor": "#ddd",
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "padding": 5,
      "position": "relative",
    }
  }
/>
`;
